cmd/storage-rest-common.go

```diff
@@ -17,7 +17,7 @@
 package cmd
 
 const (
-	storageRESTVersion       = "v11"
+	storageRESTVersion       = "v12"
```

Comment:
Why we are increasting storage version @harshavardhana ?

Reply:
Because we forgot to change it in last PR #8690